ui/src/literature/containers/LiteratureSearchContainer.jsx

```diff
@@ -206,6 +210,7 @@ const stateToProps = (state, { namespace }) => ({
   ]),
   results: state.search.getIn(['namespaces', namespace, 'results']),
   isCitationSummaryVisible: isCitationSummaryEnabled(state),
+  isCatalogerLoggedIn: isCataloger(state.user.getIn(['data', 'roles'])),
```

Comment:
```suggestion
  isCatalogerOrAdmin: isSuperUser(state.user.getIn(['data', 'roles'])) || isCataloger(state.user.getIn(['data', 'roles']))
```

Reply:
Yeah, it was my first thought too. But `isCataloger` actualy returns true if the role is `cataloger` or `superuser` as described here https://github.com/inspirehep/inspirehep/blob/master/ui/src/common/authorization.js#L18-L20